matlab/MeCorr/MATLAB_CODE_methyl.m

```diff
@@ -124,7 +121,6 @@
 glucflag1 = [ ones(1,6),  0 ,0, 0,  0];
 expval1 = [ ones(1,7),  0 , 1,  0];
 
-
 [ix, aapos] = ismember( mediareactions1(20:37),acetylation_model.rxns);
 posgluc = 1385;  % glucose uptake reaction in recon1.

```

Comment:
Don't hard-code this - use find(ismember(model.rxns, 'EX__glc_D_e')) to find the glucose exchange reaction. Same for glutamine and linoleic acid.